src/app/layout.js

```diff
@@ -33,8 +29,42 @@ export default function RootLayout({ children }) {
         <ColorSchemeScript />
       </head>
       <body
-        className={`${geistSans.variable} ${geistMono.variable} antialiased`}
+        className={`${kanit.className} min-w-[90rem] font-normal text-base bg-gray-100`}
       >
+        <div className="w-full h-20 bg-white  flex items-center px-4 pr-8 relative z-50">
+          <div className="flex-grow">
+            <div className="flex flex-row items-center gap-2">
+              <Image
+                src="/termtem-logo.svg"
+                alt="termtem logo"
+                width={56}
+                height={56}
+                priority
+              />
+              <div className="flex flex-col">
+                <p className="font-medium text-gray-600 text-lg">
+                  ระบบข้อมูลแผนที่สำหรับการบริหารจัดการภัยพิบัติ
+                  เพื่อสนับสนุนการใช้ข้อมูลห้องศูนย์บัญชาการเหตุการณ์ (EOC)
```

Comment:
Why we use different text with the figma? 
![image](https://github.com/user-attachments/assets/38acae47-c2b8-4cc7-b42a-bb005ab81ebf)

Reply:
I will change krub. Thank you 🙏